modular_skyrat/modules/mapping/code/wardrobes.dm

```diff
@@ -27,7 +27,8 @@
 					/obj/item/clothing/shoes/combat/coldres = 2)
 	premium = list(/obj/item/knife/combat/survival = 1,
 					/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 5,
-					/obj/item/clothing/gloves/combat = 3)
+					/obj/item/clothing/gloves/combat = 3,
+					/obj/item/storage/box/nif_ghost_box = 10)
```

Comment:
```suggestion
					/obj/item/storage/box/nif_ghost_box = 10,
			)
```
I can't indent properly on GH editor so pls make sure the ) is in the right spot